packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html

```diff
@@ -0,0 +1,93 @@
+<div class="upload-documents-details d-flex w-100 flex-column">
+    <h3
+        class="oui-header__title d-flex align-items-center"
+        data-ng-click="$ctrl.goBack()"
+    >
+        <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span
+        ><span
+            data-translate="{{'upload-documents-details-title-' + $ctrl.proofType}}"
+        ></span>
+    </h3>
+    <oui-tile
+        data-heading="{{:: 'upload-documents-details-choose-document' | translate }}"
+        class="my-3 w-100"
+        data-ng-if="$ctrl.numberOfDocuments > 1"
+    >
+        <oui-radio
+            data-model="$ctrl.currentDocument"
+            data-ng-repeat="(document, values) in $ctrl.proof.documents"
+            data-value="document"
+            data-on-change="$ctrl.onDocumentChange()"
+        >
+            <span
+                data-translate="{{'upload-documents-details-document-type-' + document }}"
+            ></span>
+        </oui-radio>
+    </oui-tile>
+    <oui-tile
+        data-heading="{{:: 'upload-documents-details-add-document' | translate}}"
+        class="my-3 w-100"
+    >
+        <div class="oui-box">
+            <h4
+                class="oui-box__heading"
+                data-translate="user_account_identity_documents_details_acceptance_criteria"
+            ></h4>
+            <ul>
+                <li
+                    data-ng-repeat="(index, acceptanceCriteria) in $ctrl.currentAcceptanceCriteria"
+                    data-translate="{{'upload-documents-details-acceptance-criteria-' + acceptanceCriteria }}"
+                    id="{{acceptanceCriteria + index}}"
+                ></li>
+            </ul>
+        </div>
+        <div
+            data-ng-class="'row row-cols-xs-1 row-cols-md-' + $ctrl.files.length + ' my-3'"
```

Comment:
**question:** Where are the class definitions for `row-cols-md-*` and `row-cols-xs-1`?

Reply:
Them are from bootstrap